src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -118,16 +123,14 @@ void AudioStreamAAudio::internalErrorCallback(
     if (oboeStream->wasErrorCallbackCalled()) { // block extra error callbacks
         LOGE("%s() multiple error callbacks called!", __func__);
     } else if (stream != oboeStream->getUnderlyingStream()) {
-        LOGW("%s() stream already closed or closing", __func__); // can happen if there are bugs
+        LOGW("%s() stream already closed or closing", __func__); // might happen if there are bugs
```

Comment:
The comment is a bit vague. Under what circumstances could this happen? Can we link to specific gh issues?

Reply:
I don't know of any bugs that cause this. But if it happened then it would be a bug.